### PR TITLE
Additional namespaces flag

### PR DESCRIPTION
**Pull request by marccampbell**

### CODE REVIEW

File: pkg/kotsadm/namespaces.go [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+		return nil // no error here
+	}
+
+	if gvk.Group != "kots.io" || gvk.Version != "v1beta1" || gvk.Kind != "Application" {
+		return nil
+	}
+
+	application := obj.(*kotsv1beta1.Application)
+
+	if application.Spec.AdditionalNamespaces == nil {
```
Comment (divolgin) [inline pkg/kotsadm/namespaces.go]: don't need this.  loop works with nil arrays

File: pkg/kotsadm/namespaces.go [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+		return nil
+	}
+
+	for _, additionalNamespace := range application.Spec.AdditionalNamespaces {
+		// We support "*" for additional namespaces to handle pullsecret propagation
```
Comment (divolgin) [inline pkg/kotsadm/namespaces.go]: "We `don't` support", right?

Comment (marccampbell) [reply pkg/kotsadm/namespaces.go]: We do, but we don't create that.  It (will be) handled in the kotsadm code to handle imagePulLSecret propagation

Comment (divolgin) [reply pkg/kotsadm/namespaces.go]: Does this then break on something like this:
```
 - custom_ns
 - worker*
```

Comment (marccampbell) [reply pkg/kotsadm/namespaces.go]: unsupported.  only "*" and/or specific (exact) namespaces

**Pull request merged**